contracts/WithdrawManager.sol

```diff
@@ -59,12 +74,27 @@ contract WithdrawManager {
         );
     }
 
-    function ClaimTokens(uint256 token, uint256 accountID) external {
+    function ClaimTokens(
+        uint256 token,
+        uint256 accountID,
+        uint256[4] calldata pubkey,
+        uint256[2] calldata signature,
+        bytes32[] calldata witness
+    ) external {
+        MerkleTreeUtilsLib.verifyLeaf(
+            accountRegistry.root(),
+            keccak256(abi.encodePacked(pubkey)),
+            accountID,
+            witness
+        );
+        BLS.verifySingle(
+            signature,
+            pubkey,
+            BLS.hashToPoint(APP_ID, abi.encodePacked(msg.sender))
+        );
```

Comment:
The validation has these properties:
- The pubkey must be the one associated with the accountID
- Must be signed against the same pubkey.
- The message contains `msg.sender` so that frontrunner can't steal the claim.